src/python/grpcio/grpc/__init__.py

```diff
@@ -721,6 +768,26 @@ def peer(self):
     raise NotImplementedError()
 
   @abc.abstractmethod
+  def parent(self, child=None, census=None):
+    """Creates a value to associate invoked RPCs with the current RPC.
+
+    This method's parameters have no guaranteed defaults; if either True or
+    False is desired for the value of an argument the argument should be
+    explicitly passed.
+
+    Args:
+      child: An optional boolean indicating whether or not the invoked RPC
```

Comment:
@ctiller, @jtattermusch:

1. Is census context _creation_ ever going to be a thing that needs to be supported in Python?
2. Are census context _properties_ ever going to be things that need to be supported in Python?

This draft API currently reflects "no" to both questions and treats census context as an opaque, no-features thing that comes with an incoming RPC and can either be attached or not to an outgoing RPC. If that's not what's desired, please shout now?